src/CwpSearchResult.php

```diff
@@ -65,8 +65,12 @@ public function __construct($terms = '', ArrayData $results = null)
             // Clean up the results.
             $matches = $results->Matches;
             foreach ($matches as $result) {
-                if (!$result->canView()) {
+                if (method_exists($result, 'isIndexable') && !$result->isIndexable()) {
```

Comment:
Is this a DataObject? Should it not be `$result->hasMethod('isIndexable')` so that extensions can add the method too?

Reply:
Thanks. Updated